spec.html

```diff
@@ -41517,7 +41502,7 @@ <h1>
         </h1>
```

Comment:
We should put `oldids="typedarray-species-create"` in the `emu-clause` for `TypedArrayCreateSameType` so that existing links to the old AO get redirected here.